driver/others/blas_server.c

```diff
@@ -279,6 +280,23 @@ int get_node(void);
 
 static int increased_threads = 0;
 
+#ifdef OS_LINUX
```

Comment:
Shouldn't this use the same variable as the other if-guard? I.e. `OPENBLAS_OS_LINUX` instead of `OS_LINUX`?

Reply:
The API header for OpenBLAS has OPENBLAS_ prepended to the configuration variables,
internally i.e. for building OpenBLAS itself it is just OS_LINUX etc.

Reply:
thanks for the response, learned something new again.

Reply:
come to think of it, this could probably be added to the extremely sketchy "developer manual" in the wiki